android-runtime/runtime/src/main/java/io/sdkgen/runtime/SdkgenHttpClient.kt

```diff
@@ -119,12 +119,18 @@ open class SdkgenHttpClient(
     private val random = Random()
     private val hexArray = "0123456789abcdef".toCharArray()
     private val gson = Gson()
-    private val httpClient = OkHttpClient.Builder()
-        .connectTimeout(5, TimeUnit.MINUTES)
-        .readTimeout(5, TimeUnit.MINUTES)
-        .callTimeout(5, TimeUnit.MINUTES)
-        .writeTimeout(5, TimeUnit.MINUTES)
-        .build()
+    private val logging : HttpLoggingInterceptor = HttpLoggingInterceptor().apply {
+        this.level = HttpLoggingInterceptor.Level.BODY
+        this.redactHeader("Authorization")
+        this.redactHeader("Cookie")
+    }
+    private val httpClient = OkHttpClient.Builder().apply {
+        this.addInterceptor(logging)
+        this.connectTimeout(5, TimeUnit.MINUTES)
+        this.readTimeout(5, TimeUnit.MINUTES)
+        this.callTimeout(5, TimeUnit.MINUTES)
+        this.writeTimeout(5, TimeUnit.MINUTES)
+    }.build()
```

Comment:
Why the code style changed? The other one looks fine and has less code.

Reply:
I don't have any preferences between the two, but if it remains like this, `this` before every function call is not needed

Reply:
I also don't have a certain preference between the two, but I find it more idiomatic using `apply`. The use of `this` is not really necessary.

Reply:
I think we should keep as it is currently. It's the most common way to see Builders through the internet and is the way used in [Square's documentation](https://square.github.io/okhttp/recipes/#timeouts-kt-java).